src/Identity/testassets/Identity.DefaultUI.WebSite/Properties/launchSettings.json

```diff
@@ -0,0 +1,27 @@
+{
```

Comment:
Curious, why is this file being added? I'm not opposed to it, just curious.

Reply:
VS created it after running things locally, I can remove it, I'm ambivalent, it snuck in, do you have a preference?

Reply:
Likely VS added it when someone tried to run the project.

We normally have them on all our samples/assets.

Reply:
Ok I'll keep the file then